pkg/create/templates/test.go

```diff
@@ -106,7 +114,7 @@ test_connection() {
   local result=1
   while [ $attempt -le $max_attempts ]; do
     echo "Sending GET request to http://$(container_ip):${test_port}/"
-    response_code=$(curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
+    response_code=$(docker run --rm -it sequenceiq/alpine-curl curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
```

Comment:
I see.  no i think you can leave it, but leave the original line there, commented out, with a comment about why it's being done that way, so people who don't care can easily revert the behavior.